src/main/java/nextstep/RoomEscapeApplication.java

```diff
@@ -13,12 +13,11 @@ public class RoomEscapeApplication {
     private static final String DELETE = "delete";
     private static final String QUIT = "quit";
 
+    private static ReservationDAO reservationDAO = new ReservationConsoleDAO();
```

Comment:
콘솔 어플리케이션에서도 ReservationService를 사용할 수 있도록 하여, 웹 어플리케이션과의 중복 코드를 제거해보는건 어떨까요? 😄 

```suggestion
    private ReservationService reservationService = new ReservationService(new ReservationConsoleDAO());
```